chart/kubeapps/templates/apprepository-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "kubeapps.apprepository.fullname" . }}
+  labels:
+    app: {{ template "kubeapps.apprepository.fullname" . }}
+    chart: {{ template "kubeapps.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "kubeapps.apprepository.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "kubeapps.apprepository.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ template "kubeapps.apprepository.fullname" . }}
+      containers:
+      - name: controller
+        image: "{{ .Values.apprepository.image.repository }}:{{ .Values.apprepository.image.tag }}"
```

Comment:
Maybe we should do it for consistency, but I was just going off the `helm create` template. TBH I don't really like adding the registry value, because I don't think the value needs to be so fine-grained, setting `repository=myregistry.com/apprepository-controller` works fine right?

Reply:
In other charts the image is set using a template. This could be easier down the line if we want to change the way we set the images ("registry, repository, image" or "repository,image")

Reply:
@javsalgar I couldn't find where the template was used, we seem to be just doing this: https://github.com/helm/charts/blob/master/stable/drupal/templates/deployment.yaml#L27. In any case, since this chart will eventually be going into bitnami/charts I am going to update it for consistency